Client/Info.plist

```diff
@@ -9,11 +9,11 @@
 	<key>MozDevelopmentTeam</key>
 	<string>$(DEVELOPMENT_TEAM)</string>
 	<key>LeanplumEnvironment</key>
-	<string>$(LEANPLUM_ENVIRONMENT)</string>
```

Comment:
@Sdaswani What is the problem with `$(LEANPLUM_ENVIRONMENT)` ? Or was this changed already?